.github/workflows/git-town.yml

```diff
@@ -0,0 +1,24 @@
+name: Git Town
+
+on:
+  pull_request:
+    branches:
+      - '**'
+
+concurrency:
+  group: git-town-${{ github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  git-town:
```

Comment:
Adjust this name to whatever you decide to call the file.

Reply:
Regardless of what we agree to in the previous conversation, I do agree that the name for this job could be improved! Will rename to `branch-stack` to match.